app/Resources/views/layout.html.twig

```diff
@@ -87,10 +87,10 @@
             </div>
         {% endblock alerts %}
 
-            {% block content %}
-                {% block page_content %}
-                {% endblock page_content %}
-            {% endblock content %}
+        {% block content %}
```

Comment:
Why have a content block if there is nothing in it?

Reply:
All the views are reviewed in the design branch